client/src/pages/Profile/ProfileAvailability/useStyles.tsx

```diff
@@ -0,0 +1,47 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: theme.spacing(4, 6),
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
`Grid` component already has `display: flex` by default and accepts a `direction` prop to specify `flexDirection` directly in the JSX - I would recommend to always skim through the [props](https://v4.mui.com/api/grid/#props) of a mui component to familiarize with what can be done without custom CSS.